sdk/core/azure_core_amqp/src/value.rs

```diff
@@ -36,7 +36,7 @@ impl AmqpList {
         Self(Vec::new())
     }
 
-    pub fn new_with_size(size: usize) -> Self {
+    pub fn new_with_capacity(size: usize) -> Self {
```

Comment:
Anything wrong with just `with_capacity`? It's not idiomatic to preface that with "new". `new` typically stands alone. See the hordes of `with_capacity` even in `std` for prior art.

Reply:
Nothing at all - I already fixed it after your first comment.